tests/regression/runner.py

```diff
@@ -164,7 +164,8 @@ def test_mam_send_msg(self):
         for i in range(len(response)):
             if i in pass_case:
                 res_json = json.loads(response[i]["content"])
-                self.assertTrue(valid_trytes(res_json["channel"], LEN_ADDR))
+                self.assertEqual(STATUS_CODE_200, response[i]["status_code"])
```

Comment:
The test of MAM will be implemented in another file. In addidtion, we haven't implement API for MAM yet. Could your remove the tests of MAM API?